modules/tinymce/src/themes/silver/main/ts/ui/context/ContextToolbarBounds.ts

```diff
@@ -15,8 +15,9 @@ import Editor from 'tinymce/core/api/Editor';
 import * as Settings from '../../api/Settings';
 import { UiFactoryBackstageShared } from '../../backstage/Backstage';
 
+// Note: We round the values here as we are looking for changes that aren't just 0.001px difference
 const isVerticalOverlap = (a: Bounds, b: Bounds): boolean =>
-  Math.max(a.y, b.y) <= Math.min(a.bottom, b.bottom);
+  Math.round(a.y) < Math.round(b.bottom) && Math.round(a.bottom) > Math.round(b.y);
```

Comment:
This is still going to break if for example `a.y = 0.49` and `b.bottom = 0.51`

Could we try something like
```suggestion
  b.bottom - a.y >= 1.0 && a.bottom - b.y >= 1.0
```

Alternatively, could we do a rounding / number normalising process higher up the call tree, and make sure it applies to everything. That way `0.49` and `0.51` isn't a problem, because they're treated as `0` and `1` everywhere (even on-screen)

Reply:
Yep, that we're fine with as discussed in calls/slack. We just don't want it to be picking up minimal changes such as `0.001px`.

Reply:
When I gave the thumbs up I was thinking more about the behaviour of not counting minimal changes, and not thinking so much about the particular `round` implementation. I still agree that we want to do something to avoid minimal changes, I just worry that in some cases this code won't be able to filter out minimal changes.

I don't know where we source the numbers for the two bounding boxes we're comparing, so this might be super unlikely to happen in real life, but the issue with comparing things that have come out of a `round` function is that there's always going to be a threshold where a small difference gets treated like a large difference. For example, if the boxes have a tiny overlap, but one of them is `1005.499999` and the other is `1005.500001 `, we might still end up accidentally returning true here, because one value will round down and the other will round up.

Maybe it'd be worth changing the `>= 1.0` to a different threshold, if we have a clear idea of where the line is between "this is too small a change to consider" and "this is a change worth considering as a real overlap", but I do think we need to adjust this to look at the difference between the two numbers instead of rounding them.

Reply:
Yeah fair point... okay I'll change it to ensure we meet at least the full 1px visible threshold. For reference the numbers just come from the browsers `getBoundingClientRect()` function. I won't be using `1.0` though as it can just be `1` in this case.

Reply:
Changed as discussed.